core/src/autogluon/core/models/ensemble/bagged_ensemble_model.py

```diff
@@ -60,6 +60,8 @@ class BaggedEnsembleModel(AbstractModel):
     """
 
     _oof_filename = "oof.pkl"
+    _vary_seed_across_folds: bool = True
```

Comment:
can be an `ag_args_fit` param to `BaggedEnsembleModel` with default True.

this can then be specified via `ag_args_ensemble` by the user.